agent/lib/kontena/websocket_client.rb

```diff
@@ -101,26 +100,27 @@ def send_message(msg)
       error "failed to send message: #{exc.message}"
     end
 
+    def send_notification(method, params)
```

Comment:
Is this method used anywhere? The `QueueWorker#send_message` calls `send_message(MessagePack.dump ...)` directly.

It would be nice to have all of the MessagePage protocol in one place..